integreat_cms/cms/views/users/user_form_view.py

```diff
@@ -47,6 +47,7 @@ def get(self, request, *args, **kwargs):
         """
 
         user = get_user_model().objects.filter(id=kwargs.get("user_id")).first()
+        current_user = get_user_model().objects.filter(id=request.user.id).first()
```

Comment:
```suggestion
        current_user = get_user_model().objects.filter(id=request.user.id).first()
```
This is not required. `request.user` is already the user from the database, so no reason to query it again.